assembly/src/todo/instruction/adv_ops.rs

```diff
@@ -1,13 +1,13 @@
 // CONSTANTS
 // ================================================================================================
 
-use vm_core::{code_blocks::CodeBlock, Felt, Operation::*};
+use vm_core::{code_blocks::CodeBlock, Operation::*};
 
 use crate::{todo::span_builder::SpanBuilder, AssemblerError};
```

Comment:
nit: these imports should probably go above the `CONSTANTS` line. And also probably most of them could come from `super`?